docs/man/ctags-lang-verilog.7.rst

```diff
@@ -162,6 +162,29 @@ because they cannot be overridden.
 	P3	input.sv	/^	parameter  P3 = "parameter";$/;"	c	module:no_parameter_port_list	parameter:
 	L7	input.sv	/^	localparam L7 = "localparam";$/;"	c	module:no_parameter_port_list
 
+Supported Roles
+~~~~~~~~~~~~~~~
+
+.. warning::
+
+   The support for references in Universal Ctags is still
+   experimental; the names of the roles can be changed in the future.
+
+
+.. code-block:: console
+
+	$ ./ctags --list-roles=SystemVerilog
+	#KIND(L/N) NAME ENABLED DESCRIPTION
+	m/module   decl on      declaring instances
+
+	$ ./ctags --list-roles=Verilog
+	#KIND(L/N) NAME ENABLED DESCRIPTION
+	m/module   decl on      declaring instances
+
+The parsers extract modules names used in instance declarations as
+reference tags. ``decl`` is the role for the names. See "TAG ENTRIES"
+section of :ref:`ctags(1) <ctags(1)>` about reference tags and roles.
+
```

Comment:
This is not the output from ctags-lang-verilog.7.rst.in. 

https://app.circleci.com/pipelines/github/universal-ctags/ctags/2663/workflows/33fb3bc0-056e-48b8-bab8-694279a28c98/jobs/28427

This is a true fail.

Reply:
Updated. Thank you.